ckanext/harvest/queue.py

```diff
@@ -68,27 +68,25 @@ def get_connection_redis():
 
 
 def get_gather_queue_name():
-    return 'ckan.harvest.{0}.gather'.format(config.get('ckan.site_id',
-                                                       'default'))
+    return 'ckanext-harvest:{0}:gather'.format(
```

Comment:
Changing the queue name seems like bad news for existing users that upgrade to this version of ckanext-harvest. Can't we keep it the same, for ease?